scripts/ai/jobs/CpAIJobCombineUnloader.lua

```diff
@@ -83,6 +82,7 @@ end
 
 --- Called when parameters change, scan field
 function CpAIJobCombineUnloader:validate(farmId)
```

Comment:
@pvaiko Not sure why you didn't use the same setup, as with the bale finder?

Reply:
Not sure what you mean. With the unloader, we want two positions: 
- one **outside** of the field where the Giants AI should drive it, and where it waits for the combine
- one on the field, which we are not driving to, only use it to mark the field we are working on. 
So to me, this is different than the bale finder.

Reply:
The same two positions are used for the balefinder, when the mode is started from the GUI.
The only difference might be the position of the giants marker.

The only meaningful difference between balefinder and unloader is the starting from the HUD, as we don't want to start on the field(balefinder) but in front of it. I've corrected this for the unloader, so the hud isn't reseting the position.